sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -53,8 +55,12 @@ public TimeSpan MaxAutoRenewDuration
             }
         }
 
-        /// <summary>Gets or sets the time to wait for receiving a message.</summary>
-        /// <value>The time to wait for receiving the message.</value>
+        /// <summary>Gets or sets the timeout to wait for receiving a message. If a session is running out of active messages, it will wait up 
+        /// to this value to close the current session. Once the session is closed, the session lock will be lost and can no longer receive new messages. 
+        /// This value has an impact on the message throughput. If the value is very large, then every time when the traffic in a session is slowing down, 
```

Comment:
Lets try to split this into `summary` and `remarks`.
Impact on throughput is not a summary, and can be moved to remarks